snap/info_snap_yaml.go

```diff
@@ -80,6 +80,8 @@ type appYaml struct {
 
 	After  []string `yaml:"after,omitempty"`
 	Before []string `yaml:"before,omitempty"`
+
+	WatchdogTimeout uint `yaml:"watchdog,omitempty"`
```

Comment:
This needs to be _watchdog-timeout_ to align with stop-timeout and friends, and the type needs to be timeout.Timeout, same as StopTimeout too, so that it parses correctly.